GVFS/GVFS.Common/Git/GitAuthentication.cs

```diff
@@ -152,6 +166,25 @@ public bool TryInitializeAndRequireAuth(ITracer tracer, out string errorMessage)
             return false;
         }
 
+        public void SetupSslIfNeeded(ITracer tracer, HttpClientHandler httpClientHandler, GitProcess gitProcess)
```

Comment:
Naming suggestion: 

When I first read `SetupSslIfNeeded` I thought this method was doing something to setup `this.GitSsl`.  Assuming my understanding is correct it actually looks like the method configures ssl on the `HttpClientHandler`.

If that's the case consider renaming this to something like `ConfigureHttpClientHandlerSSLIfNeeded`.  If my understanding is not correct, please point me in the right direction 😄

Reply:
Sure, will make the name more explicit. Just FYI, this exact name was suggested by one of the other reviewers, so I hope he doesn't come back asking me to revert it.